EIPS/eip-5071.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 5071
+title: NFT Token Acceptance Standard
+author: Mihir Wadekar (@mw2000)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-05
+requires: 165
+---
+
+## Simple Summary
+
+A standardized way to retrieve the types of ERC-20 tokens that are accepted by a particular NFT collection for buying and selling in a NFT Marketplace.
+
```

Comment:
Not a valid EIP section (anymore).  Please see https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for the proper current structure of an EIP.
```suggestion
```